larray/core/axis.py

```diff
@@ -2312,7 +2312,16 @@ def _translate_axis_key(self, axis_key, bool_passthrough=True):
         """
         from .array import LArray
 
+        # Need to convert string keys to groups otherwise command like
+        # >>> ndtest((5, 5)).drop('1[a0]')
+        # will work although it shouldn't
+        if isinstance(axis_key, basestring):
+            key = _to_key(axis_key)
+            if isinstance(key, Group):
+                axis_key = key
+
         if isinstance(axis_key, ExprNode):
+            # ALD: Exception('yada') --> What?
```

Comment:
Oops, this obviously should never have been committed. ;-). This was a test to see if this code path was actually used. It seems like either it is not used, or there is no test exercising it... The idea was to check which option it is before either removing the code or adding tests... and it seems like I forgot to do that.
FWIW, the same comment applies a few lines below for the other "yada"

Reply:
Do I remove these lines then?

Reply:
no, it would pollute your PR ;-). And I'd like to actually check the situation before removing them. I've opened an issue to avoid forgetting it: #711.